crates/re_data_source/src/load_file.rs

```diff
@@ -190,36 +206,40 @@ pub(crate) fn load(
                             contents,
                             tx_loader,
                         ) {
-                            re_log::error!(?path, loader = loader.name(), %err, "Failed to load data from file");
+                            if err.is_not_supported() {
+                                return;
+                            }
+                            re_log::error!(?path, loader = loader.name(), %err, "Failed to load data");
                         }
                     } else if let Err(err) =
                         loader.load_from_path(store_id, path.clone(), tx_loader)
                     {
+                        if err.is_not_supported() {
+                            return;
+                        }
                         re_log::error!(?path, loader = loader.name(), %err, "Failed to load data from file");
                     }
-                }
-            });
 
-            #[cfg(target_arch = "wasm32")]
-            spawn(|| {
-                if let Some(contents) = contents.as_deref() {
-                    let contents = Cow::Borrowed(contents);
+                    re_log::debug!(loader = loader.name(), ?path, "compatible loader found");
+                    tx_feedback.send(()).ok();
+                });
+            }
 
-                    if let Err(err) =
-                        loader.load_from_file_contents(store_id, path.clone(), contents, tx_loader)
-                    {
-                        re_log::error!(?path, loader = loader.name(), %err, "Failed to load data from file");
-                    }
-                }
-            });
-        }
+            drop(tx_feedback);
+
+            rx_feedback.recv().is_ok() // at least one compatible loader was found
```

Comment:
Again, using a named type would be helpful here

```suggestion
            rx_feedback.recv() == Ok(CompatibleLoaderFound)
```